SIPS/sip-27.md

```diff
@@ -0,0 +1,38 @@
+---
+sip: 27
+title: Add new crypto Synths
+status: Proposed
+author: Garth Travers (@garthtravers)
+discussions-to: (https://discordapp.com/invite/CDTvjHY)
+
+created: 2019-11-20
+---
+
+<!--You can leave these HTML comments in your merged SIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SIPs. Note that an SIP number will be assigned by an editor. When opening a pull request to submit your SIP, please use an abbreviated title in the filename, `sip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the SIP.-->
+Add new long and inverse Synths for three new cryptocurrencies: LINK, XRP, and LTC. 
```

Comment:
[np] worth adding in their full names as well

Reply:
Done